src/schema/basic_classes.yaml

```diff
@@ -464,15 +423,8 @@ classes:
           interpolated: true
       execution_resource:
         required: true
-      part_of:
```

Comment:
@eecavanna  and I were wondering what would become of the `part_of` slot. @eecavanna we will have to write a migrator that removes the `part_of` slot from the `WorkflowExecution` subclasses. @aclum we don't need to retain any of the values that are currently populated in the `part_of` slot do we? I think I just replaced the `part_of` values that currently exist with the id of the `WorkflowChain` instance in the `WorkflowChain` migrator.

Reply:
Right now part_of is inconsistently populated with either a self reference, that is it is the same value as id, OR it is populated with the DataGeneration id. If we wanted to keep the values that are currently populated with `part_of` the action would be to if ( the value of part_of == id) then delete, elsif (part_of value matches nmdc:(omprc|dgms|dgns)) then rename part_of to [associated_studies](https://microbiomedata.github.io/berkeley-schema-fy24/associated_studies/). If we want to do that I need to change this PR. 

 **I am in favor of not retaining any of the values currently populated in the `part_of` slot as it is redundant information.**
 
 With this PR `WorkflowChain` doesn't exist anymore so a `WorkflowChain` migrator wouldn't exist.